zotero_master/translators/ubtue_Studia_Religiologica.js

```diff
@@ -56,25 +56,28 @@ function getSearchResults(doc, checkOnly) {
   return found ? items : false;
 }
 
-function getAbstractAndKeywords(item) {
+function extractVolumeIssue(doc, item, volumeIssueEntry) {
+	var issueVolume = volumeIssueEntry.match(/tom\s+(\d+)(?:.*numer\s+(\d+))?/i);
+	item.volume = issueVolume[1];
+	item.issue = '';
+	if (typeof issueVolume[2] != 'undefined') {
+    	item.issue = issueVolume[2];
+	}
+}
+
+function getAbstractAndKeywords(item, doc) {
```

Comment:
If you switch the language to Polish, it does not work. You have to fix the regex e.g. volume in english site.

Reply:
Lieber @socheres ,
vielen Dank, es funktioniert jetzt in beiden Sprachen und auch bei den Bänden, bei denen Voumel statt Volume steht.
Viele Grüße, Helena Nebel